### DISCUSSION TITLE
Are we merging with the new version?

### DISCUSSION THREAD

**Discussion by kaneryu** [z-huang/music#1309] [General]
(this is for @z-huang)
I think most users have switched over to @Malopieds's fork of Innertune, and i've noticed that you've made some new commits. Are you planning to try and merge with his version?

**Comment by z-huang** [comment]
I will only merge some of his fixes (very appreciated that he solved some bugs that's very tricky). We will have different UI, though.

**Reply by NeeRaj-2401** [reply]
> We will have different UI, though.

3rd time

**Reply by Malopieds** [reply]
That's totally understandable that you don't want to have some changes, but I will try to make a "migration" script for the users that are using my fork and want to switch back to yours!

If you ever need help/think that you don't have time to work on some issues, I will gladly try to help on some commit on this repo !

**Reply by z-huang** [reply]
@Malopieds many thanks!

**Reply by amitness** [reply]
@z-huang Could you please add a "select all" button in downloaded page? I would like to export all my downloaded songs to a playlist so that it can be exported/imported easily. This is available in the @Malopieds fork.

**Comment by kaneryu** [comment]
@z-huang > We will have different UI, though. 

do your current UI or a new one different from that?

**Reply by z-huang** [reply]
Current UI, maybe with some modifications in the future.

**Comment by 7heMech** [comment]
On this topic @Malopieds will your fork integrate changes from this one?

**Reply by Malopieds** [reply]
Yep, I will try as long as it's not conflicting too much with my previous changes.
